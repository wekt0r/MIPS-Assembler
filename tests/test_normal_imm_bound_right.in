addi $s0, $s1, -32768
addi $s1, $s2, 32767
