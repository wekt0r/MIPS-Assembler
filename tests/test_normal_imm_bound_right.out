.text
00000000	22308000	addi	$s0,$s1,-32768
00000004	22517FFF	addi	$s1,$s2,32767
